proto/mettle.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package build.please.remote.mettle;
+
+
+service Bootstrap {
+
+    rpc ServeExecutions(ServeExecutionsRequest) returns (ServeExecutionsResponse);
+}
+
+message Job {
+    string ID = 1;
+    bytes Current = 2;
```

Comment:
Ahahaha ok this is how you got around compiling the third-party protos. Seems legit.

FYI there is also `google.protobuf.Any` which is a _slightly_ nicer version of the same, but this is fine for what we need.

Reply:
haha this would solve all my problems but importing that throw a "Import "google/protobuf/any.proto" was not found or had errors." I've just tested the same thing on core and it imports fine so not sure what the issue is.

Reply:
This looks like https://github.com/thought-machine/please/issues/1641 (funny timing...)

As discussed, let's not worry for now. I don't think it would make a significant difference really.